lib/main.coffee

```diff
@@ -31,9 +26,20 @@ module.exports =
     @subscriptions.add atom.config.observe 'linter-phpmd.rulesets',
       (rulesets) =>
         @rulesets = rulesets
-    @subscriptions.add atom.config.observe 'linter-phpmd.projectRules',
-      (projectRules) =>
-        @projectRules = projectRules
+    if atom.config.get('linter-phpmd.projectRules') == true &&
+    atom.config.get('linter-phpmd.rulesets') ==
+    atom.config.defaultSettings['linter-phpmd'].rulesets
```

Comment:
It's so easy to find awesome little shortcuts like that when debugging and just start using them without checking the API, I've done it before :stuck_out_tongue:.